src/test/resources/tya/lapsedRevised.json

```diff
@@ -0,0 +1,71 @@
+{
+  "appeal": {
+    "caseReference": "SC333/33/33333",
+    "appealNumber": "abcde12345",
+    "benefitType": "pip",
+    "name": "Mr. C Charlie",
+    "surname": "Charlie",
+    "status": "LAPSED_REVISED",
+    "latestEvents": [
+      {
+        "date": "2017-03-02T17:27:50.297Z",
+        "type": "LAPSED_REVISED",
+        "contentKey": "status.lapsedRevised"
+      }
+    ],
+    "historicalEvents": [
+      {
+        "decisionLetterReceiveByDate": "2017-03-10T17:27:50.297Z",
+        "date": "2017-03-01T17:27:50.297Z",
+        "type": "HEARING",
+        "contentKey": "status.hearing"
+      },
+      {
+        "decisionLetterReceiveByDate": "2017-02-22T12:50:11.437Z",
+        "date": "2017-02-11T12:50:11.437Z",
+        "type": "DORMANT",
+        "contentKey": "status.dormant"
+      },
+      {
+        "evidenceType": "Evidence received",
+        "evidenceProvidedBy": "Evidence received",
+        "date": "2016-12-30T00:00:00.000Z",
+        "type": "EVIDENCE_RECEIVED",
+        "contentKey": "status.evidenceReceived"
+      },
+      {
+        "hearingContactDate": "2016-12-15T15:11:45.827Z",
+        "date": "2016-11-03T15:11:45.827Z",
+        "type": "DWP_RESPOND",
+        "contentKey": "status.dwpRespond"
+      },
+      {
+        "dwpResponseDate": "2016-11-03T08:32:44.633Z",
+        "date": "2016-09-29T08:32:44.633Z",
+        "type": "APPEAL_RECEIVED",
+        "contentKey": "status.appealReceived"
+      },
+      {
+        "postcode": "L2 5UZ",
+        "venueName": "Prudential House",
+        "hearingDateTime": "2017-07-17T23:00:00.000Z",
+        "addressLine1": "36 Dale Street",
+        "addressLine2": "",
+        "addressLine3": "Liverpool",
+        "googleMapUrl": "https://www.google.com/maps/place/36+Dale+Street+Liverpool+Merseyside+L2+5UZ/@53.407820,-2.988509",
+        "date": "2016-01-20T15:15:06.223Z",
+        "type": "HEARING_BOOKED",
+        "contentKey": "status.hearingBooked"
+      }
+    ],
+    "regionalProcessingCenter": {
+      "name":"LIVERPOOL",
+      "addressLines":["HM Courts & Tribunals Service","Social Security & Child Support Appeals",
+        "Prudential Buildings","36 Dale Street"],
+      "city":"LIVERPOOL",
+      "postcode":"L2 5UZ",
+      "phoneNumber":"0300 123 1142",
+      "faxNumber":"0870 324 0109"
+    }
+  }
+}
```

Comment:
Santhosh, this test data events are not in correct order. DORMANT is the last status in an appeal.